frontends/bmd/src/pages/user_settings_modal.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useContext, useEffect } from 'react';
+
+import { Button, Prose, Modal } from '@votingworks/ui';
+import { BallotContext } from '../contexts/ballot_context';
+import { SettingsTextSize } from '../components/settings_text_size';
+import { handleGamepadKeyboardEvent } from '../lib/gamepad';
+import { DEFAULT_USER_SETTINGS } from '../config/globals';
+
+export function VoterSettingsModal(): JSX.Element {
+  const { userSettings, setUserSettings } = useContext(BallotContext);
+
+  function closeUserSettingsModal() {
+    return setUserSettings({ showSettingsModal: false });
+  }
+
+  function resetUserSettingsToDefaults() {
+    setUserSettings(DEFAULT_USER_SETTINGS);
+  }
+
+  useEffect(() => {
+    function handleKeyDown(event: KeyboardEvent) {
+      if (event.key === 'ArrowRight') {
+        event.stopPropagation();
+        handleGamepadKeyboardEvent({ ...event, key: 'ArrowDown' });
+      }
+      if (event.key === 'ArrowLeft') {
+        event.stopPropagation();
+        handleGamepadKeyboardEvent({ ...event, key: 'ArrowUp' });
+      }
+    }
+    document.addEventListener('keydown', handleKeyDown, { capture: true });
+    return () =>
+      document.removeEventListener('keydown', handleKeyDown, { capture: true });
+  }, []);
```

Comment:
Could you ELI5 why this is necessary?

Reply:
In the Settings modal, pressing the right and left arrows navigates to the next/previous contests behind the settings modal. The settings modal UI has a bit of horizontal UI, so we could do one of two things:
- map the left/right to the up/down so that it works as a user might expect (as implemented)
- disable the left/right when the settings modal is option (might be a better solution)

Reply:
This seems plenty good, just wondering.